seedfarmer/commands/_stack_commands.py

```diff
@@ -313,6 +327,14 @@ def deploy_module_stack(
 
     _logger.debug("project_managed_policy_output is : %s", stack_outputs)
     if project_managed_policy_stack_exists:
+        if stack_outputs.get("StackStatus") and "_IN_PROGRESS" in stack_outputs.get("StackStatus"):
+            _logger.info("The managed policy stack is not complete, waiting 60 seconds")
+            time.sleep(60)
```

Comment:
as discussed offline, its good to have this refactored with retry logic, while sleeping for 30 seconds